test/models/item_test.rb

```diff
@@ -3,8 +3,44 @@
 class ItemTest < ActiveSupport::TestCase
   should validate_presence_of(:title)
   should validate_presence_of(:image_path)
-  should validate_presence_of(:artist_id)
+  should validate_presence_of(:user_id)
   should validate_presence_of(:category_id)
   should validate_presence_of(:price)
   should validate_presence_of(:description)
+
+  test "item can belong to an artist" do
+    artist = create(:artist)
+    category = create(:category)
+    artist.items.create(title: 'Title',
+                        description: 'Description',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.